README.md

```diff
@@ -137,7 +137,7 @@ Once you have earned the Responsive Web Design, Algorithms and Data Structures,
 
 #### Legacy Certifications
 
-We also have 4 legacy certifications dating back to our 2015 curriculum, which are still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
+We also have 4 legacy certifications dating back to our 2015 curriculum, which is still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
```

Comment:
This also changes the reading of the sentence. Before it inferred that the legacy certifications are still available, now it suggests that the *2015 curriculum* is still available.

Reply:
Yes, I agree with @nhcarrigan.  Upon second reading, it should be "are" since it is related to "4 legacy certifications".